README.rst

```diff
@@ -0,0 +1,183 @@
+.. |DiscordLogo| image:: https://img.icons8.com/color/48/000000/discord-logo.png
+   :target: https://discordapp.com
+
+|DiscordLogo| OH-Bot
+======================================
+
+The Discord Office Hour Butler
```

Comment:
This was the tagline for the old bot, I think `The "Better" Office Hours Bot` would be more appropriate.